vercel.json

```diff
@@ -2,7 +2,7 @@
   "functions": {
     "api/*.js": {
       "memory": 128,
-      "maxDuration": 30
+      "maxDuration": 10
```

Comment:
I was trying to check why it still gives me error here.
Anyways I will make it **30** again